lib/hammer_cli_foreman_puppet/version.rb

```diff
@@ -1,5 +1,5 @@
 module HammerCLIForemanPuppet
   def self.version
-    @version ||= Gem::Version.new '0.0.1'
+    @version ||= Gem::Version.new '0.0.2'
```

Comment:
This should probably be a separate commit when you release and tag the new version, you can push it directly